test-data/unit/fine-grained-modules.test

```diff
@@ -1588,6 +1588,39 @@ x = ''
 ==
 ==
 
+[case testImportStarSomethingMoved1]
+[file p.py]
```

Comment:
You never import `p` so I am not sure anything is actually checked in this test.

Reply:
`testfinegrained` picks up all the files involved rather than following imports. I'll add an import though to make it more clear, since most tests still do that.